identity/aziot-identityd-config/src/lib.rs

```diff
@@ -180,13 +183,9 @@ mod tests {
 
         match s.provisioning.provisioning {
```

Comment:
I guess you could make these two `if !matches!(...) { panic!(...); }` to shorten them even more.

Reply:
huh, I'm surprised `clippy` didn't catch that - it's usually pretty good about linting against this pattern.